azure-pipelines.yml

```diff
@@ -193,6 +194,7 @@ stages:
         git submodule update --init
       displayName: 'Fetch submodules'
     - script: |
+            set -e
```

Comment:
If we do this, why not going all the way to `set -euxo pipefail`

https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/

Reply:
The `set -x` part writes to `stderr` which makes the CI run fail. We can also allow `stderr` output, but that risks warnings being missed.

Reply:
Sure the -x is not really needed as already done by default by the CI.